dashboard/app/views/levels/editors/fields/_aitutor_fields.html.haml

```diff
@@ -1,5 +1,10 @@
 %h1.control-legend{data: {toggle: "collapse", target: "#ai_tutor"}}
   AI Tutor
 #ai_tutor.in.collapse
-  .field
-    = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :ai_tutor_available, description: "AI Tutor Available"}
+  
+  = f.label :ai_tutor_level_specific_prompt, 'Level-Specific Prompt'
+  %p Prompt used to specify the behavior of AI Tutor on a per level basis.
+  = f.text_area :ai_tutor_level_specific_prompt, placeholder: 'Write your level-specific prompt here. If left blank, this level will use the default system prompt.', rows: 4, class:"input-block-level"
```

Comment:
Should we say what the default system prompt is? Right now it's "'You are a tutor in a high school classroom where the students are learning Java using the Code.org curriculum. Answer their questions in plain, easy-to-understand English. Do not write any code. Do not answer the question if it is not about Java or computer programming."  

I think it would be totally reasonable to punt on this for now, but might be a nice-to-have down the road depending on how things evolve. One advantage of re-writing these fields in React, if we have time/inclination etc, would be that we could pull in the base prompts from aiTutor/constants.ts to display for context.